src/Integration/Exclusions/ExclusionSettingsFileStorage.cs

```diff
@@ -56,6 +56,18 @@ public ExclusionSettingsFileStorage(IConfigurationProvider bindingConfigProvider
 
         public void SaveSettings(ServerExclusions settings)
         {
+            var bindingConfiguration = bindingConfigProvider.GetConfiguration();
```

Comment:
Based on Ugra's PR with one difference: exceptions are thrown to the caller. This would allow the binding process to abort.